### PR TITLE
Task 1409890: Refresh of .NET article, View certificates with MMC

### LINKED ISSUE #7586: Outdated MMC steps
The steps to view certificates in the MMC snap-in are slightly incorrect, at least based on my version of Management Console (3.0). Step 4 wasn't necessary, the dialog box in step 5 was different and step 9 wasn't required. I didn't follow any of the optional steps nor the IE instructions so I'm not sure if they're up to date or not.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f950f479-532b-ab38-a676-653495777197
* Version Independent ID: 30fa2c1a-86bd-e683-3e6f-8a268f4dc4e5
* Content: [How to: View Certificates with the MMC Snap-in](https://docs.microsoft.com/en-us/dotnet/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in#feedback)
* Content Source: [docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md](https://github.com/dotnet/docs/blob/master/docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md)
* Product: **dotnet-framework**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by dksimpson**
## Summary

Task 1409890: https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1409890/

Verify content freshness and update procedures.

Fixes #7586

### CODE REVIEW

**Comment by dksimpson** [comment]
Hi @rpetrusha, I've finished my updates - ready for your review.

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 1]
```diff
@@ -1,55 +1,80 @@
...
-  
-1.  In Internet Explorer, click **Tools**, then click **Internet Options** to display the **Internet Options** dialog box.  
-  
-2.  Click the **Content** tab.  
-  
-3.  Under **Certificates**, click **Certificates**.  
-  
-4.  To view details of any certificate, select the certificate and click **View**.  
+# How to: View certificates with the MMC snap-in
+When you create a secure client or service, you can use a [certificate](working-with-certificates.md) as the credential. For example, a common type of credential is the X.509 certificate, which you create with the <xref:System.ServiceModel.Security.X509CertificateInitiatorClientCredential.SetCertificate%2A> method.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
When you create a secure client or service, you can use a [certificate](working-with-certificates.md) as the credential. For example, a common type of credential is the X.509 certificate, which you create with the <xref:System.ServiceModel.Security.X509CertificateInitiatorClientCredential.SetCertificate%2A?displayProperty=nameWithType> method. 
```

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 2]
```diff
@@ -1,55 +1,80 @@
...
-  
-2.  Click the **Content** tab.  
-  
-3.  Under **Certificates**, click **Certificates**.  
-  
-4.  To view details of any certificate, select the certificate and click **View**.  
+# How to: View certificates with the MMC snap-in
+When you create a secure client or service, you can use a [certificate](working-with-certificates.md) as the credential. For example, a common type of credential is the X.509 certificate, which you create with the <xref:System.ServiceModel.Security.X509CertificateInitiatorClientCredential.SetCertificate%2A> method. 
+
+There are three different types of certificate stores that you can examine with the Microsoft Management Console (MMC):
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
There are three different types of certificate stores that you can examine with the Microsoft Management Console (MMC) on Windows systems:
```

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 3]
```diff
@@ -1,55 +1,80 @@
...
+
+- Local computer: Store is local to the device and global to all users on the device.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
- Local computer: The store is local to the device and global to all users on the device.
```

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 4]
```diff
@@ -1,55 +1,80 @@
...
+
+- Current user: Store is local to the current user account on the device.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
- Current user: The store is local to the current user account on the device.
```

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 5]
```diff
@@ -1,55 +1,80 @@
...
+
+- Service account: Store is local to a particular service on the device.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
- Service account: The store is local to a particular service on the device.
```

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 6]
```diff
@@ -1,55 +1,80 @@
...
+
+  
+## To view certificates in the MMC snap-in
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
## View certificates in the MMC snap-in 
```

File: docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md [hunk 7]
```diff
@@ -1,55 +1,80 @@
...
+  
+2. To view your certificates, under **Certificates - Local Computer** in the left pane, expand the directory for the type of certificate you want to view.
+
+### To view certificates for the current user
+
+1. Select **Run** from the **Start** menu, and then enter *certmgr.msc*. 
+
+    The Certificate Manager tool for the current user appears. 
+  
+2. To view your certificates, under **Certificates - Current User** in the left pane, expand the directory for the type of certificate you want to view.
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/how-to-view-certificates-with-the-mmc-snap-in.md]:
```suggestion
2. To view your certificates, under **Certificates - Current User** in the left pane, and expand the directory for the type of certificate you want to view.
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for updating this article, @dksimpson. I've left a number of suggested changes.

**Review by Lxiamail — APPROVED** [review]
With Ron's suggestion, I have no additional feedback.

**Comment by dksimpson** [comment]
Hi @rpetrusha, I've incorporated your review comments - ready to merge.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks for the additional changes, @dksimpson. I'll merge your PR now.

**Pull request merged**